site/src/pages/release-notes/1.9.0.mdx

```diff
@@ -0,0 +1,237 @@
+---
+date: 2021-06-23
+---
+
+## 🌟 New features
+
+- You can now bind [GraphQL Java](https://www.graphql-java.com/) to <type://ServerBuilder> 
+  using <type://GraphqlService>. #3318 #3373
+  ```java
+  ServerBuilder sb = ...;
+  sb.service("/graphql", GraphqlService.builder()
+                                       .schemaFile(schemaFiles)
+                                       .runtimeWiring(w -> ...)
+                                       .configureDataLoaderRegistry(registery -> ...)
+                                       ...
+                                       .build());
+  ```
+  <Tip>
+
+  `armeria-graphql` is currently experimental, so you should be careful using the feature.
+
+  </Tip>
+- You can now transform the objects of <type://HttpRequest> and <type://HttpResponse> using a function.
+  #3606 #3624
+  ```java
+  HttpRequest req1 = ...;
+  HttpRequest req2 = req1.mapHeaders(headers -> ...)
+                         .mapData(data -> ...)
+                         .mapTrailers(trailers -> ...);
```

Comment:
How about adding a little bit more concrete example than just `...`, at least for the lambda expressions?